<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{qlineargradient(spread:repeat, x1:0.495, y1:1, x2:0.49, y2:0, stop:0 rgba(242, 21, 141, 255), stop:1 rgba(255, 255, 255, 255))
}
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Download Files</string>
     </attribute>
     <widget class="QProgressBar" name="QProBar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnStart">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnBrs">
      <property name="geometry">
       <rect>
        <x>414</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtUrl">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtLoc">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DownLoad Youtube Video</string>
     </attribute>
     <widget class="QPushButton" name="QBtnStart_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnBrs_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtUrl_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Youtube URL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtLoc_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Quality</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnBrs_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="QProBar_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Download Youtube Playlist</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>location</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Quality</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtLoc_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="QTxtUrl_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Youtube URL</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnStart_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnBrs_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>current video </string>
      </property>
     </widget>
     <widget class="QProgressBar" name="QProBar_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="QBtnBrs_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
